transcom/mymove
# B-20438 and B-20440

## Agility Tickets
## [B-20440](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20440)
## [B-20438](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20438)

## Previous PRs (into INT) Associated with this PR
| &nbsp; &nbsp; &nbsp; &nbsp; PR &nbsp; &nbsp; &nbsp; &nbsp; | Definition |
| --------------- | --------------- |
| [13571](https://github.com/transcom/mymove/pull/13571) | Corrected the package.json from pointing directly to the branch to point at the tag (#v1.2.4) of the react-file-viewer |
| [13543](https://github.com/transcom/mymove/pull/13543) | This is in response to I-13228 and I-13225 (kickbacks) |
| [13520](https://github.com/transcom/mymove/pull/13520) | Due to I-13228 issue the previous merge-commit for these tickets had to be reverted |
| [13508](https://github.com/transcom/mymove/pull/13508) | Resolved an issue with missing QueryClientProvider in tests |
| [13487](https://github.com/transcom/mymove/pull/13487) | Initial merge to allow for rotation/zoom and save capabilities for images associated with docs |

## Verification that it all works
If you would like to verify that rotation/zoom still works on this branch, make sure you perform the following steps to ensure it will work as designed.

1. Delete the node_modules folder
```
rm -rf node_modules
```
2. Clear Yarn cache
```
yarn cache clean
```
3. Re-install Yarn modules
```
yarn install
```
4. Start the server and client
5. After creating a move with an upload (image)
6. As Office user (SC if counseling is needed.. TOO/TIO/Prime if not), navigate to the move and view the uploads/documents/orders and verify that you are able to rotate the images, zoom in and out on the images, and save the rotation.
